### PR TITLE
🌐 [i18n-KO] Translated `tasks/masked_language_modeling.mdx` to Korean

### PR DESCRIPTION
**Pull request by HanNayeoniee**
# What does this PR do?

Translated the `tasks/masked_language_modeling.mdx` file of the documentation to Korean.
Thank you in advance for your review. 

Part of https://github.com/huggingface/transformers/issues/20179

## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [x] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review?

Team PseudoLab, may you please review this PR?
@0525hhgus, @KIHOON71, @sim-so, @gabrielwithappy, @HanNayeoniee, @wonhyeongseo, @jungnerd

May you please review this PR?
@sgugger, @ArthurZucker, @eunseojo

### CODE REVIEW

**Review by wonhyeongseo — CHANGES_REQUESTED** [review]
~L189 리뷰 완료
고생 많으셨습니당. 작업 속도가 엄청 빨라지셨군요. 다음에도 리뷰해볼게요.
행복한 하루 보내시기 바래요 ...φ(・∀・*)

File: docs/source/ko/tasks/masked_language_modeling.mdx [hunk 1]
```diff
@@ -0,0 +1,450 @@
...
+각 문장을 개별적으로 토큰화하는 대신 리스트를 문자열로 변환하여 한번에 토큰화할 수 있습니다.
+
+다음은 각 예제에 대해 문자열로 이루어진 리스트를 `join`하고 결과를 토큰화하는 첫 번째 전처리 함수입니다:
+
+```py
+>>> def preprocess_function(examples):
+...     return tokenizer([" ".join(x) for x in examples["answers.text"]])
+```
+
+이 전처리 함수를 전체 데이터 세드에 적용하기 위해 🤗 Datasets [`~datasets.Dataset.map`] 메소드를 사용합니다.
```
Comment (sim-so) [inline docs/source/ko/tasks/masked_language_modeling.mdx:140]:
```suggestion
이 전처리 함수를 전체 데이터 세트에 적용하기 위해 🤗 Datasets [`~datasets.Dataset.map`] 메소드를 사용합니다. 
```

File: docs/source/ko/tasks/masked_language_modeling.mdx [hunk 2]
```diff
@@ -0,0 +1,450 @@
...
+```
+
+전체 데이터 세트에 `group_texts` 함수를 적용합니다:
+
+```py
+>>> lm_dataset = tokenized_eli5.map(group_texts, batched=True, num_proc=4)
+```
+
+이제 [`DataCollatorForLanguageModeling`]을 사용하여 데이터 예제의 배치를 생성합니다. 
+데이터 세트 전체를 최대 길이로 패딩하는 것 보다 collation 단계에서 매 배치 안에서의 최대 길이로 문장을 *동적으로 패딩*하는 것이 더 효율적입니다.
```
Comment (sim-so) [inline docs/source/ko/tasks/masked_language_modeling.mdx:188]:
```suggestion
데이터 세트 전체를 최대 길이로 패딩하는 것보다 collation 단계에서 매 배치 안에서의 최대 길이로 문장을 *동적으로 패딩*하는 것이 더 효율적입니다.
```

**Review by sim-so — CHANGES_REQUESTED** [review]
고생 많으셨습니다!
몇 가지 철자만 수정 제안 드립니다 😄

**Review by 0525hhgus — COMMENTED** [review]
좋은 번역 감사합니다 😄 저는 현재 번역 진행된 부분까지 모두 괜찮으며, 이전 리뷰어분들의 의견에 동의합니다 👍

File: docs/source/ko/tasks/masked_language_modeling.mdx [hunk 3]
```diff
@@ -0,0 +1,450 @@
...
+
+# 마스크드 언어 모델링(Masked language modeling)[[masked_language_modeling]]
+
+[[open-in-colab]]
+
+<Youtube id="mqElG5QJWUg"/>
+
+마스크드 언어 모델링은 시퀀스에서 마스크드 토큰을 예측하며, 모델은 양방향으로 토큰에 액세스할 수 있습니다.
+즉, 모델은 토큰의 왼쪽과 오른쪽 양쪽에서 접근할 수 있습니다.
+마스크드 언어 모델링은 전체 시퀀스에 대한 문맥적 이해가 필요한 작업에 적합하며, BERT가 그 예에 해당합니다.
```
[the remark this answers is not in the document]
Comment (HanNayeoniee) [reply docs/source/ko/tasks/masked_language_modeling.mdx]: [위키독스-딥러닝을 이용한 자연어처리 입문](https://wikidocs.net/153992)을 보고 Masked Language Model을 `마스크드 언어 모델링`으로 번역했는데, 어색해서 `마스킹된 언어 모델링`으로 변경하였습니다.

File: docs/source/ko/tasks/masked_language_modeling.mdx [hunk 4]
```diff
@@ -0,0 +1,450 @@
...
+
+전체 데이터 세트에 `group_texts` 함수를 적용합니다:
+
+```py
+>>> lm_dataset = tokenized_eli5.map(group_texts, batched=True, num_proc=4)
+```
+
+이제 [`DataCollatorForLanguageModeling`]을 사용하여 데이터 예제의 배치를 생성합니다. 
+데이터 세트 전체를 최대 길이로 패딩하는 것 보다 collation 단계에서 매 배치 안에서의 최대 길이로 문장을 *동적으로 패딩*하는 것이 더 효율적입니다.
+
```
[the remark this answers is not in the document]
Comment (HanNayeoniee) [reply docs/source/ko/tasks/masked_language_modeling.mdx:189]: 원형님의 빠른 리뷰 속도에 놀랐습니다 🫢 감사합니다!

File: docs/source/ko/_toctree.yml [hunk 5]
```diff
@@ -57,10 +57,10 @@
         title: (번역중) Question answering
       - local: in_translation
         title: (번역중) Causal language modeling
+      - local: tasks/masked_language_modeling
+        title: 마스크드 언어 모델링(Masked language modeling)
       - local: in_translation
-        title: (번역중) Masked language modeling
```
[the remark this answers is not in the document]
Comment (HanNayeoniee) [reply docs/source/ko/_toctree.yml:61]: `_toctree.yml` 파일이 아닌 `.mdx` 파일의 목차 부분이 문제였습니다!
L13 [[ ]] 안에 dash(-)를 사용해야하는데, underscore(_)을 써서 빌드가 안 됐었습니다.

@gabrielwithappy님, 도와주셔서 감사합니다 :)

File: docs/source/ko/tasks/masked_language_modeling.mdx [hunk 6]
```diff
@@ -0,0 +1,443 @@
...
+...     args=training_args,
+...     train_dataset=lm_dataset["train"],
+...     eval_dataset=lm_dataset["test"],
+...     data_collator=data_collator,
+... )
+
+>>> trainer.train()
+```
+
+Once training is completed, use the [`~transformers.Trainer.evaluate`] method to evaluate your model and get its perplexity:
```
Comment (gabrielwithappy) [inline docs/source/ko/tasks/masked_language_modeling.mdx:253]: 영어 부분이 아직 남아 있는 것 같습니다.

File: docs/source/ko/_toctree.yml (hunk 5 above)
[the remark this answers is not in the document]
Comment (HanNayeoniee) [reply docs/source/ko/_toctree.yml:61]: 다른 부분에도 문제가 있는지 빌드가 안 되었네용... 내일 다시 확인해볼게요

[the remark this answers is not in the document]
Comment (HanNayeoniee) [reply docs/source/ko/_toctree.yml]: 해결했습니다 감사합니다 :)

**Review by sgugger — APPROVED** [review]
Thanks for your contribution!

**Review by ArthurZucker — COMMENTED** [review]
A very small nit 😉 Otherwise good job 🔥
